### PR TITLE
My new branch

### PR DESCRIPTION
**Pull request by madhur1808**
link of google form and instagram handle will open in new tab for better user experience.

### CODE REVIEW

File: contributors/contributorsList.js [hunk 1]
```diff
@@ -4,4 +4,9 @@ contributors = [
     fullname: "Ashish",
     username: "https://github.com/ashishthehulk",
   },
+  {
+    id: 2,
+    fullname: "Ashish",
```
Comment (robogyan-adgitm) [inline contributors/contributorsList.js]: Name?

Comment (madhur1808) [reply contributors/contributorsList.js]: shutikshan madhur

**Review by robogyan-adgitm — CHANGES_REQUESTED** [review]
Add your name in the list bruh :rofl:

**Comment by robogyan-adgitm** [comment]
resolve conflicts @Madhur1808

**Review by robogyan-adgitm — APPROVED** [review]

**Pull request merged**